projects/components/src/confirmation/confirmation-modal.component.scss

```diff
@@ -14,7 +14,7 @@
   .description {
     @include body-1-regular($gray-7);
     text-align: center;
-    word-break: break-all;
+    word-break: break-word;
```

Comment:
Any side effects of this change? Possibly on API DNA - mark sensitive data side? 

cc @jake-bassett

Reply:
I think it should be fine. Tested at few places. Looked alright. We are just going from breaking mid-word to a more readable word break.

Reply:
Fine if you tested it.